### ISSUE TITLE
does not works on FireFox 45

### ISSUE THREAD

**Issue by Cyrille37** [stefanocudini/leaflet-compass#5]
Hi
Perhaps `this._divcompass.style.webkitTransform = "rotate(" + angle + "deg)";` is not available in FireFox.

nota: it works with Chromium 48.

**Comment by Cyrille37** [comment]
That's it:

```
        if( typeof this._divcompass.style.webkitTransform != "undefined" ){
            this._divcompass.style.webkitTransform = "rotate(" + angle + "deg)";
        } else if( typeof this._divcompass.style.transform != "undefined" ) {
            this._divcompass.style.transform = "rotate(" + angle + "deg)";
        }
```

**Comment by stefanocudini** [comment]
solved from version 1.0.0
